src/matchers/toBeString/index.js

```diff
@@ -0,0 +1,28 @@
+import { matcherHint, printExpected, printReceived } from 'jest-matcher-utils';
+
+import predicate from './predicate';
+
+const passMessage = received => () =>
+  matcherHint('.not.toBeString', 'received', '') +
+  '\n\n' +
+  'Expected value to not be a string received:\n' +
+  `  ${printReceived(received)}`;
+
+const failMessage = received => () =>
+  matcherHint('.toBeString', 'received', '') +
+  '\n\n' +
+  'Expected value to be a string:\n' +
```

Comment:
This error message isn't quite right. I think it should be something like the passMessage above:

```js
const failMessage = received => () =>
  matcherHint('.toBeString', 'received', '') +
  '\n\n' +
  'Expected value to be a string received:\n' +
  `  ${printReceived(received)}`;
```